google-cloud-bigquery-storage/README.md

```diff
@@ -0,0 +1,71 @@
+# Ruby Client for the BigQuery Storage API
+
+google-cloud-bigquery-storage is the official library for the BigQuery Storage API.
+
+The BigQuery Storage API provides fast access to BigQuery managed storage.
+
+Actual client classes for the various versions of this API are defined in
+_versioned_ client gems, with names of the form `google-cloud-bigquery-storage-v*`.
+The gem `google-cloud-bigquery-storage` is a convenience wrapper library that brings the
+verisoned gems in as dependencies, and provides high-level methods for
+constructing clients.
+
+View the [Client Library Documentation](https://googleapis.dev/ruby/google-cloud-bigquery-storage/latest)
+for this library, google-cloud-bigquery-storage, to see the convenience methods for
+constructing client objects. Reference documentation for the client objects
+themselves can be found in the client library documentation for the versioned
+client gems:
+[google-cloud-bigquery-storage-v1](https://googleapis.dev/ruby/google-cloud-bigquery-storage-v1/latest).
+
+See also the [Product Documentation](https://cloud.google.com/bigquery/docs/reference/storage)
+for more usage information.
+
+## Quick Start
+
+```
+$ gem install google-cloud-bigquery-storage
+```
+
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+1. [Enable the API.](https://console.cloud.google.com/apis/library/bigquerystorage.googleapis.com)
+1. {file:AUTHENTICATION.md Set up authentication.}
```

Comment:
@dazuma Will this link be generated once the file is in master branch? Right now if I view the rendered file, I still see: `{file:AUTHENTICATION.md Set up authentication.}`.

Reply:
This is YARD syntax. When you view the YARD-generated page, it still shows the text?

Reply:
@quartzmo I pulled your branch, ran yard, and the resulting rendered webpage includes the link as expected. It's not going to show up as such in github, but I'm betting that Rubyists are more likely to stumble on the yardoc rendering of this internal readme than on the github page.